packages/main/src/Menu.js

```diff
@@ -0,0 +1,490 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import {
+	isLeft,
+	isRight,
+	isEnter,
+} from "@ui5/webcomponents-base/dist/Keys.js";
+import {
+	isPhone,
+	isTablet,
+	isDesktop,
+} from "@ui5/webcomponents-base/dist/Device.js";
+import {
+	getI18nBundle,
+} from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import ResponsivePopover from "./ResponsivePopover.js";
+import Button from "./Button.js";
+import List from "./List.js";
+import StandardListItem from "./StandardListItem.js";
+import Icon from "./Icon.js";
+
+import staticAreaMenuTemplate from "./generated/templates/MenuTemplate.lit.js";
+
+import {
+	MENU_BACK_BUTTON_ARIA_LABEL,
+	MENU_CLOSE_BUTTON_ARIA_LABEL,
+} from "./generated/i18n/i18n-defaults.js";
+
+// Styles
+import staticAreaMenuCss from "./generated/themes/Menu.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-menu",
+	properties: /** @lends sap.ui.webcomponents.main.Menu.prototype */ {
+		/**
+		 * Defines the title of the menu (displayed on mobile).
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		 title: {
```

Comment:
"title" is HTML attribute and we can't declare properties with the same name. We can only use "title" property as part of the metadata in case we want to forward the html "title" priority se externally to specific  ShadowDOM node. "headerText' seems more consistent
![Screenshot 2022-04-04 at 10 41 18](https://user-images.githubusercontent.com/15702139/161496718-a5973ad8-b503-4c0a-aec0-0d359d704020.png)

Reply:
Done